packages/math-input/src/index.ts

```diff
@@ -5,10 +5,7 @@
 import "../less/main.less";
 
 export {default as KeypadInput} from "./components/input/math-input";
-export {
-    keypadConfigurationPropType,
-    keypadElementPropType,
-} from "./components/prop-types";
+export {keypadElementPropType} from "./components/prop-types";
```

Comment:
API change. Should this be a major/minor changeset?

Reply:
Major, but I asked https://khanacademy.slack.com/archives/CEA6W0F6F/p1681488884159309

Reply:
And the answer is: it's a major version bump!